src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -937,4 +934,32 @@
   <data name="ColorPicker_Editor.Text" xml:space="preserve">
     <value>Editor</value>
   </data>
+  <data name="PowerLauncher_Plugins.Text" xml:space="preserve">
+    <value>Plug-ins</value>
+  </data>
+  <data name="PowerLauncher_ActionKeyword.Text" xml:space="preserve">
+    <value>Direct activation phrase</value>
+  </data>
+  <data name="PowerLauncher_AuthoredBy.Text" xml:space="preserve">
+    <value>Authored by</value>
+    <comment>example: Authored by Microsoft</comment>
+  </data>
+  <data name="PowerLauncher_IncludeInGlobalResult.Content" xml:space="preserve">
+    <value>Include in global result</value>
+  </data>
+  <data name="PowerLauncher_EnablePluginToggle.AutomationProperties.Name" xml:space="preserve">
+    <value>Enable plugin</value>
+  </data>
+  <data name="Run_AdditionalOptions.Text" xml:space="preserve">
+    <value>Additional options</value>
+  </data>
+  <data name="Run_NotAccessibleWarning.Text" xml:space="preserve">
+    <value>Please define an activation phrase or allow this plugin for the global results to use it</value>

```

Comment:
```suggestion
    <value>Please define an activation phrase or allow this plugin for the global results to use it.</value>
```

We agreed that all warnings should end with a dot.

Reply:
Can you provide a link to the discussion, please? Warning on General and File Explorer pages do not have dots at the end.

Reply:
> Can you provide a link to the discussion, please? 

https://github.com/microsoft/PowerToys/pull/9838#discussion_r580461737

> Warning on General and File Explorer pages do not have dots at the end.

I had mentioned this too and asked @niels9001 if he can fix this. Don't got a reaction.

Reply:
@htcfreek Please create a new issue so we don't need to handle it here. I can pick up the work there.

Reply:
> @htcfreek Please create a new issue so we don't need to handle it here. I can pick up the work there.

Done: #9881